### ISSUE TITLE
Screen reader is announcing questions as toggle button off for every question which is confusing to the screen reader users.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#4892]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users will be confused on interacting with the questions if they are being announced as toggle buttons.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://servicesuat.microsoft.com/#/opdashboard/

### How to reproduce the issue?

1.	Hit the URL https://servicesuat.microsoft.com/#/opdashboard/
2.	Tab till iris bot icon in the header section and press enter.
3.	Iris bit welcome screen will be displayed.
4.	Tab till top questions for your role and press enter.
5.	Start Narrator and tab till first question and observe the announcement.

### What do you expect?

Screen reader should announce as button as announcing toggle button can be confusing to the screen reader users.

### What actually happened?

Screen reader is announcing questions as toggle button off for every question which is confusing to the screen reader users.

### Do you have any screenshots or recordings to repro the issue?

[11059099A11y_Intelligent Support_Iris bot_Top questions for your role_ScreenReader.webm](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/d41c2196-484f-4e6b-8804-0f078e76d29a)

### MAS reference

MAS 1.3.1 – Info and Relationships

**Comment by v-prahu** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#Benchmark;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
This bug is copied from TCS team: [Bug 11059099](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11059099)

**Comment by compulim** [comment]
This is by design.

Adaptive Cards supports repetitive submissions. In other words, after a button is clicked, the user can go back and click another button. This is the design of Adaptive Cards.

Thus, multiple choices requires us to use a push button approach via `aria-pressed="true"` to indicate what button is selected.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/753c0d46-6b9b-4375-be1b-e7cd1b20c00d)

If repetitive submission is not desirable, the customer should consider using suggested actions, instead of Adaptive Cards.
